server/routes/v1/upload.ts

```diff
@@ -109,7 +110,10 @@ export const postUpload = [
           })
         }
       }
-      req.log.info({ versionId: version._id }, 'Created model version')
+      req.log.info(
+        { version: _.pick(version, ['_id', 'version', 'metadata.highLevelDetails.name', 'model']) },
```

Comment:
Not needed anymore with new serialization method.

Reply:
Didn't spot this! Changed